gradle.properties

```diff
@@ -33,7 +33,7 @@ kovenantVersion=3.3.0
 lifecycleVersion=2.5.1
 materialVersion=1.8.0
 mockitoKotlinVersion=4.1.0
-okhttpVersion=3.12.1
+okhttpVersion=5.0.0-alpha.14
```

Comment:
Hey is there any reason to use alpha version? Can we opt for a stable version instead?

Reply:
Rolled back to stable `4.12.0` version of okhttp